style.css

```diff
@@ -0,0 +1,17 @@
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+
+.item {
+  width: 100%;
+  position: relative;
```

Comment:
Given that you're utilizing a random image pulled from Unsplash, I'd suggest utilizing styling that will ensure coverage of the whole background. I know for me personally, I kept getting images that did not fill the space adequately based on their individual image specifications.